docs/docs/tutorial/chapter2/side-quest.md

```diff
@@ -4,7 +4,7 @@ Redwood likes GraphQL. We think it's the API of the future. Our GraphQL implemen
 
 ![Redwood Data Flow](https://user-images.githubusercontent.com/300/75402679-50bdd180-58ba-11ea-92c9-bb5a5f4da659.png)
 
-The front-end uses [Apollo Client](https://www.apollographql.com/docs/react/) to create a GraphQL payload sent to [GraphQL Yoga](https://www.graphql-yoga.com) and [Envelop](https://www.envelop.dev/docs), for which that `graphql.{js,ts}` file acts as the entry-point to.
+The front-end uses [Apollo Client](https://www.apollographql.com/docs/react/) to create a GraphQL payload sent to [GraphQL Yoga](https://www.graphql-yoga.com) and [Envelop](https://www.envelop.dev/docs), for which that `graphql.{js,ts}` file acts as the entry-point.
```

Comment:
```suggestion
The front-end uses [Apollo Client](https://www.apollographql.com/docs/react/) to create a GraphQL payload sent to [GraphQL Yoga](https://www.graphql-yoga.com) and [Envelop](https://www.envelop.dev/docs), which that `graphql.{js,ts}` file acts as the entry-point to.
```

I'm a bit torn here, but I think I find this way of writing it a little easier to read.

But I'm not a native English speaker, so I'll leave the decision for someone else. @jtoar, what do you think?

Reply:
From my side, both are totally fine. Some say "ending with a preposition is wrong" but they're wrong, see [Marriam Webster's post](https://www.merriam-webster.com/words-at-play/prepositions-ending-a-sentence-with).

I would personally prefer "entry-point to" over "for which"

Reply:
> I would personally prefer "entry-point to" over "for which"

Great! Let's get this merged then 🙂